release/tablelist.js

```diff
@@ -4457,11 +4462,22 @@ angular
 
         options.start = options.start || moment().startOf('month').format("YYYY-MM-DD");
         options.end = options.end || moment().endOf('month').format("YYYY-MM-DD");
-        options.ticket = options.ticket || 'false';
 
         return Venue.listInventory(options).$promise;
       };
 
+      VenueService.prototype.listActiveInventory = function(venueId, options) {
+        
+        if (!venueId) throw new Error('venueId is required');
+        if (!options) throw new Error('options.required');
+
+        options.id = venueId;
+        options.start = options.start || moment().startOf('month').unix();
```

Comment:
Probably don't need default here. Especially since we are forcing the caller to pass in an options object which should have start / end values if the caller wants them.

If we were to pass defaults we probably want it more like from today to next week instead of a whole month (which could be asking for a whole bunch of past dates)